config/routes.rb

```diff
@@ -4,6 +4,11 @@
 
   # Defines the root path route ("/")
   resources :users, only: [:index,  :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+    resources :comments, only: [:create]
```

Comment:
- [ ] I couldn't find `:new` route for comments. Kindly add `:new` route/and view the file for it. The comments controller should have new and create actions.

There should be a form in the views/comments/new.html.erb file which accepts input for creating comments.

Reply:
done